### ISSUE TITLE
I want to translate the article to chinese!

### ISSUE THREAD

**Issue by chenxiaoleizi** [greggman/threejsfundamentals#23]
Hi! I am a `three.js` developer, i had read the article, really like it, it is really good! I think it is a good resource for `three.js` learner! So i want to translate it to chinese.

**Comment by greggman** [comment]
That would be great! There are some instructions in the [README](https://github.com/greggman/threejsfundamentals/blob/master/README.md). Do you need anything else to get started?

**Comment by chenxiaoleizi** [comment]
Should i fork the project or clone the project directly?

**Comment by greggman** [comment]
If you're going to do it through github you should click fork, then clone it to your computer from your fork, then start adding translations in `threejs/lessons/zh_cn`, check them in, push to your fork, make a PR request

**Comment by chenxiaoleizi** [comment]
ok! thanks! I am going to start.

**Comment by xiaoluoHe** [comment]
@chenxiaoleizi 
Hello, are you still translating? I have translated `Three.js Primitives` while learning the articles.
So, may I join your work?

**Comment by greggman** [comment]
Translations are super welcome!!!  Please feel free to submit pull requests with new translations. Thank you!

**Comment by a775691134** [comment]
Can I participate in the translation? My English is not very good, and I want to learn English in translation. I can do it a little bit. Can I translate one article and submit a pr?

**Comment by a775691134** [comment]
And I also want to send the translated articles and the original text to my blog and some technical websites. I will explain the source, okay?

**Comment by greggman** [comment]
yes of course. please translate and feel free to post on your blog. just
please keep the license as is

**Comment by Roy-Kid** [comment]
大佬好. 我想参与到翻译工作中, 请问目前这件事有谁来负责, 现在的计划是什么? 我们是不是应该有一个非常熟悉图形学和threejs的人负责对翻译件的质量进行把关?
Hi, everyone, I want to join the translation job. I want to ask who is in charge of the Chinese translation. I think we should select a guru to control the quality of the translation script.

**Comment by greggman** [comment]
There is no one in charge really. I would suggest if you want to participate you just choose an article and declare here that you are working on it.

**Comment by Roy-Kid** [comment]
I start to translate _Camera_, _Rendering On Demand_, _Optimizing Lots of Objects_ and _Optimizing Lots of Objects Animated_ chapters. I will finish it within one week if my tutor doesn't push me in the research.
